import sys
import pymysql
from PyQt5 import QtCore, QtGui, QtWidgets


class menu_UI(object):
    def setupUi(self, Menu, firstName, lastName, phoneNumber, emailAddress, ID, age, ssn, weight, height, creditCardNumber, billingAmount, insuranceNumber, medicationList, appointmentDates, startTimes, endTimes, appointmentIDs, cur, conn):
        
        Menu.setObjectName("Menu")
        Menu.resize(800, 600)
        self.centralWidget = QtWidgets.QWidget(Menu)
        self.centralWidget.setObjectName("centralWidget")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.centralWidget)
        self.horizontalLayout.setContentsMargins(11, 11, 11, 11)
        self.horizontalLayout.setSpacing(6)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.tabWidget = QtWidgets.QTabWidget(self.centralWidget)
        self.tabWidget.setObjectName("tabWidget")
        self.tab_1 = QtWidgets.QWidget()
        self.tab_1.setObjectName("tab_1")
        self.gridLayout = QtWidgets.QGridLayout(self.tab_1)
        self.gridLayout.setContentsMargins(11, 11, 11, 11)
        self.gridLayout.setSpacing(6)
        self.gridLayout.setObjectName("gridLayout")
        self.lineEdit_1 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_1.setObjectName("lineEdit_1")
        self.gridLayout.addWidget(self.lineEdit_1, 2, 1, 1, 1)
        self.label_4 = QtWidgets.QLabel(self.tab_1)
        self.label_4.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_4.setObjectName("label_4")
        self.gridLayout.addWidget(self.label_4, 4, 2, 1, 1)
        self.lineEdit_9 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_9.setObjectName("lineEdit_9")
        self.gridLayout.addWidget(self.lineEdit_9, 10, 1, 1, 1)
        self.lineEdit_2 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.gridLayout.addWidget(self.lineEdit_2, 2, 3, 1, 1)
        self.lineEdit_3 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.gridLayout.addWidget(self.lineEdit_3, 4, 1, 1, 1)
        self.lineEdit_7 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_7.setObjectName("lineEdit_7")
        self.gridLayout.addWidget(self.lineEdit_7, 8, 1, 1, 1)
        self.lineEdit_8 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_8.setObjectName("lineEdit_8")
        self.gridLayout.addWidget(self.lineEdit_8, 8, 3, 1, 1)
        self.lineEdit_5 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_5.setObjectName("lineEdit_5")
        self.gridLayout.addWidget(self.lineEdit_5, 6, 1, 1, 1)
        self.lineEdit_10 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_10.setObjectName("lineEdit_10")
        self.gridLayout.addWidget(self.lineEdit_10, 10, 3, 1, 1)
        self.label_3 = QtWidgets.QLabel(self.tab_1)
        self.label_3.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_3.setObjectName("label_3")
        self.gridLayout.addWidget(self.label_3, 4, 0, 1, 1)
        self.label_1 = QtWidgets.QLabel(self.tab_1)
        self.label_1.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_1.setObjectName("label_1")
        self.gridLayout.addWidget(self.label_1, 2, 0, 1, 1)
        self.lineEdit_4 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_4.setObjectName("lineEdit_4")
        self.gridLayout.addWidget(self.lineEdit_4, 4, 3, 1, 1)
        self.lineEdit_6 = QtWidgets.QLineEdit(self.tab_1)
        self.lineEdit_6.setObjectName("lineEdit_6")
        self.gridLayout.addWidget(self.lineEdit_6, 6, 3, 1, 1)
        self.label_7 = QtWidgets.QLabel(self.tab_1)
        self.label_7.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_7.setObjectName("label_7")
        self.gridLayout.addWidget(self.label_7, 8, 0, 1, 1)
        self.label_2 = QtWidgets.QLabel(self.tab_1)
        self.label_2.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_2.setObjectName("label_2")
        self.gridLayout.addWidget(self.label_2, 2, 2, 1, 1)
        self.label_5 = QtWidgets.QLabel(self.tab_1)
        self.label_5.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_5.setObjectName("label_5")
        self.gridLayout.addWidget(self.label_5, 6, 0, 1, 1)
        self.label_6 = QtWidgets.QLabel(self.tab_1)
        self.label_6.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_6.setObjectName("label_6")
        self.gridLayout.addWidget(self.label_6, 6, 2, 1, 1)
        self.label_8 = QtWidgets.QLabel(self.tab_1)
        self.label_8.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_8.setObjectName("label_8")
        self.gridLayout.addWidget(self.label_8, 8, 2, 1, 1)
        self.label_9 = QtWidgets.QLabel(self.tab_1)
        self.label_9.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_9.setObjectName("label_9")
        self.gridLayout.addWidget(self.label_9, 10, 0, 1, 1)
        self.label_10 = QtWidgets.QLabel(self.tab_1)
        self.label_10.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_10.setObjectName("label_10")
        self.gridLayout.addWidget(self.label_10, 10, 2, 1, 1)
        self.tabWidget.addTab(self.tab_1, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.tab_2)
        self.gridLayout_2.setContentsMargins(11, 11, 11, 11)
        self.gridLayout_2.setSpacing(6)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.dateTimeEdit_5 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_5.setObjectName("dateTimeEdit_5")
        self.gridLayout_2.addWidget(self.dateTimeEdit_5, 2, 1, 1, 1)
        self.dateTimeEdit_1 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_1.setObjectName("dateTimeEdit_1")
        self.gridLayout_2.addWidget(self.dateTimeEdit_1, 2, 0, 1, 1)
        self.dateTimeEdit_7 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_7.setObjectName("dateTimeEdit_7")
        self.gridLayout_2.addWidget(self.dateTimeEdit_7, 4, 1, 1, 1)
        self.dateTimeEdit_3 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_3.setObjectName("dateTimeEdit_3")
        self.gridLayout_2.addWidget(self.dateTimeEdit_3, 4, 0, 1, 1)
        self.dateTimeEdit_2 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_2.setObjectName("dateTimeEdit_2")
        self.gridLayout_2.addWidget(self.dateTimeEdit_2, 3, 0, 1, 1)
        self.dateTimeEdit_4 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_4.setObjectName("dateTimeEdit_4")
        self.gridLayout_2.addWidget(self.dateTimeEdit_4, 5, 0, 1, 1)
        self.dateTimeEdit_8 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_8.setObjectName("dateTimeEdit_8")
        self.gridLayout_2.addWidget(self.dateTimeEdit_8, 5, 1, 1, 1)
        self.dateTimeEdit_6 = QtWidgets.QDateTimeEdit(self.tab_2)
        self.dateTimeEdit_6.setObjectName("dateTimeEdit_6")
        self.gridLayout_2.addWidget(self.dateTimeEdit_6, 3, 1, 1, 1)
        self.groupBox_2 = QtWidgets.QGroupBox(self.tab_2)
        self.groupBox_2.setObjectName("groupBox_2")
        self.gridLayout_4 = QtWidgets.QGridLayout(self.groupBox_2)
        self.gridLayout_4.setContentsMargins(11, 11, 11, 11)
        self.gridLayout_4.setSpacing(6)
        self.gridLayout_4.setObjectName("gridLayout_4")
        self.gridLayout_3 = QtWidgets.QGridLayout()
        self.gridLayout_3.setSpacing(6)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.calendarWidget_2 = QtWidgets.QCalendarWidget(self.groupBox_2)
        self.calendarWidget_2.setObjectName("calendarWidget_2")
        self.gridLayout_3.addWidget(self.calendarWidget_2, 0, 0, 1, 1)
        self.gridLayout_4.addLayout(self.gridLayout_3, 0, 0, 1, 1)
        self.gridLayout_2.addWidget(self.groupBox_2, 0, 0, 1, 2)
        self.label_11 = QtWidgets.QLabel(self.tab_2)
        self.label_11.setMaximumSize(QtCore.QSize(16777215, 50))
        self.label_11.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 18pt \"Lucida Calligraphy\";")
        self.label_11.setObjectName("label_11")
        self.gridLayout_2.addWidget(self.label_11, 1, 0, 1, 1)
        self.tabWidget.addTab(self.tab_2, "")
        self.tab_3 = QtWidgets.QWidget()
        self.tab_3.setObjectName("tab_3")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.tab_3)
        self.verticalLayout.setContentsMargins(11, 11, 11, 11)
        self.verticalLayout.setSpacing(6)
        self.verticalLayout.setObjectName("verticalLayout")
        self.label_12 = QtWidgets.QLabel(self.tab_3)
        self.label_12.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_12.setObjectName("label_12")
        self.verticalLayout.addWidget(self.label_12)
        self.lineEdit_12 = QtWidgets.QLineEdit(self.tab_3)
        self.lineEdit_12.setMaximumSize(QtCore.QSize(300, 50))
        self.lineEdit_12.setObjectName("lineEdit_12")
        self.verticalLayout.addWidget(self.lineEdit_12)
        self.label_13 = QtWidgets.QLabel(self.tab_3)
        self.label_13.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_13.setObjectName("label_13")
        self.verticalLayout.addWidget(self.label_13)
        self.lineEdit_13 = QtWidgets.QLineEdit(self.tab_3)
        self.lineEdit_13.setMaximumSize(QtCore.QSize(300, 50))
        self.lineEdit_13.setObjectName("lineEdit_13")
        self.verticalLayout.addWidget(self.lineEdit_13)
        self.label_14 = QtWidgets.QLabel(self.tab_3)
        self.label_14.setStyleSheet("color: rgb(46, 125, 132);\n"
"font-weight: bold;\n"
"font: 12pt \"Lucida Calligraphy\";")
        self.label_14.setObjectName("label_14")
        self.verticalLayout.addWidget(self.label_14)
        self.lineEdit_14 = QtWidgets.QLineEdit(self.tab_3)
        self.lineEdit_14.setMaximumSize(QtCore.QSize(300, 50))
        self.lineEdit_14.setObjectName("lineEdit_14")
        self.verticalLayout.addWidget(self.lineEdit_14)
        self.tabWidget.addTab(self.tab_3, "")
        self.horizontalLayout.addWidget(self.tabWidget)
        Menu.setCentralWidget(self.centralWidget)
        self.menuBar = QtWidgets.QMenuBar(Menu)
        self.menuBar.setGeometry(QtCore.QRect(0, 0, 800, 22))
        self.menuBar.setObjectName("menuBar")
        Menu.setMenuBar(self.menuBar)
        self.mainToolBar = QtWidgets.QToolBar(Menu)
        self.mainToolBar.setObjectName("mainToolBar")
        Menu.addToolBar(QtCore.Qt.TopToolBarArea, self.mainToolBar)
        self.statusBar = QtWidgets.QStatusBar(Menu)
        self.statusBar.setObjectName("statusBar")
        Menu.setStatusBar(self.statusBar)

        self.retranslateUi(Menu)
        self.tabWidget.setCurrentIndex(2)
        QtCore.QMetaObject.connectSlotsByName(Menu)

    def retranslateUi(self, Menu):
        _translate = QtCore.QCoreApplication.translate
        Menu.setWindowTitle(_translate("Menu", "Menu"))
        self.label_4.setText(_translate("Menu", "Email Address:"))
        self.label_3.setText(_translate("Menu", "Phone Number:"))
        self.label_1.setText(_translate("Menu", "First Name:"))
        self.label_7.setText(_translate("Menu", "Weight:"))
        self.label_2.setText(_translate("Menu", "Last Name:"))
        self.label_5.setText(_translate("Menu", "ID:"))
        self.label_6.setText(_translate("Menu", "SSN:"))
        self.label_8.setText(_translate("Menu", "Height:"))
        self.label_9.setText(_translate("Menu", "Age:"))
        self.label_10.setText(_translate("Menu", "Medication List:"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_1), _translate("Menu", "Profile"))
        self.groupBox_2.setTitle(_translate("Menu", "Calendar"))
        self.label_11.setText(_translate("Menu", "Appointments:"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_2), _translate("Menu", "Appointment"))
        self.label_12.setText(_translate("Menu", "Insurance Number:"))
        self.label_13.setText(_translate("Menu", "Credit Card Number:"))
        self.label_14.setText(_translate("Menu", "Billing Amount:"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_3), _translate("Menu", "Billing Info"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Menu = QtWidgets.QMainWindow()
    ui = Ui_Menu()
    ui.setupUi(Menu)
    Menu.show()
    sys.exit(app.exec_())

